packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.Props.ts

```diff
@@ -75,4 +75,12 @@ export interface ITooltipHostProps extends React.HTMLAttributes<HTMLDivElement |
    * Optional class name to apply to tooltip host.
    */
   hostClassName?: string;
+
+  /**
+   * Manual way to display tooltip when the element is not directly focusable but an ancestor is,
+   * so it can pass down that state via this property.
+   * It honors overflow rules and keyboard and mouse interaction, so this just adds another interaction way
+   * delegated on another element.
+   */
+  isFocused?: boolean;
```

Comment:
For that sake, I would use a regular Tooltip and set visibility via that prop.
Instead, I want to use the overflow mode from this `TooltipHost`. Also I want to use the mouse-over behavior to show and hide the tooltip. I'm just not reusing the focus/blur behavior because my `TooltipHost` is not focusable, so I'm providing that info from the parent element that is actually focusable. So my behavior is much closer to `TooltipHost`, both conceptually- and implementation-wise, and I can see how others will search this functionality here.